misc-tools/dj_make_chroot.in

```diff
@@ -159,7 +159,7 @@ fi
 if [ "$DISTRO" = 'Ubuntu' ]; then
 
 # Packages to include during bootstrap process (comma separated):
-INCLUDEDEBS="software-properties-common"
+INCLUDEDEBS=""
 
 # Packages to install after upgrade (space separated):
 INSTALLDEBS="default-jre-headless pypy python3 locales"
```

Comment:
If you're removing, should pypy still be there and should python3->pypy3?

Reply:
Good question. We might want to replace both with pypy3 indeed, but let's leave that for a different PR.